tests/playwright/src/specs/extension-installation-smoke.spec.ts

```diff
@@ -134,10 +134,19 @@ for (const { extensionName, extensionType } of extentionTypes) {
         await playExpect(extensionDetailsPage.status).toBeVisible({ timeout: 15000 });
       });
 
-      test('Extension is active', async () => {
+      test('Extension is active and there are not errors', async () => {
         const extensionsPage = await navigationBar.openExtensions();
         const extensionPage = await extensionsPage.openExtensionDetails(extensionName, extensionLabel, extensionType);
+        await playExpect(extensionPage.header).toBeVisible();
```

Comment:
See this is exactly what I was talking about adding complexity that is not required, this needs to be:
```suggestion
        await playExpect(extensionPage.heading).toBeVisible();
```

Because there are two fields named `header` and `heading` nobody knows which one to use, header is just the general header, it will return true wherever  the navigation is located, because there is a header everywhere, it's `heading` that validates we are on the correct page.

` this.header = page.getByRole('region', { name: 'Header' });`
` this.heading = this.header.getByRole('heading', { name: extensionName });`

Reply:
mistake, thanks.